src/bionics.cpp

```diff
@@ -958,28 +948,10 @@ bool player::install_bionics( const itype &type, int skill_level, bool autodoc )
 
     const bionic_id &bioid = type.bionic->id;
     const int difficult = type.bionic->difficulty;
-    int chance_of_success;
-    if( skill_level != -1 ) { // NPC surgeon installation
-        chance_of_success = bionic_manip_cos( skill_level,
-                                              skill_level,
-                                              skill_level,
-                                              skill_level,
-                                              difficult, autodoc );
-    } else if( autodoc ) { // Autodoc installation
-        ///\EFFECT_INT increases chance of success installing bionics with unspecified skill level
-        chance_of_success = bionic_manip_cos( int_cur,
-                                              get_skill_level( skilll_firstaid ),
-                                              get_skill_level( skilll_computer ),
-                                              get_skill_level( skilll_electronics ),
-                                              difficult, autodoc );
-    } else { // Manual self-installation
-        ///\EFFECT_INT increases chance of success installing bionics with unspecified skill level
-        chance_of_success = bionic_manip_cos( int_cur,
-                                              get_skill_level( skilll_electronics ),
-                                              get_skill_level( skilll_firstaid ),
-                                              get_skill_level( skilll_mechanics ),
-                                              difficult, autodoc );
-    }
+    int chance_of_success = installer.bionic_manip_cos( skilll_firstaid,
```

Comment:
The same here. NPC surgeons are using unspecified skill set for installing, manual installing uses different skill set, not the same as while using autodoc.